src/App/VisibilityBar.tsx

```diff
@@ -32,13 +33,49 @@ const VisibilityBar: FC<{ isSidePanelEnabled: boolean }> = ({
     const isLogVisible = useSelector(isLogVisibleSelector);
     const autoScroll = useSelector(autoScrollSelector);
 
+    useHotKey({
+        hotKey: 'shift+m',
```

Comment:
You are using a wild mix of modifiers (sometimes shift, sometimes alt, sometimes ctrl). Especially using shift alone as a modifier for a shortcut is rather uncommon (shift+m alone usually just means “Capital M”). Could you maybe limit this to using only ctrl and alt?

Reply:
Discussed this with @ketile, who said that it's not really a problem. We will add shortcuts in the apps tomorrow, so any suggestions on which shortcuts bind to which actions is appreciated 👍 

It's difficult to make shortcuts for "Save", "Select" and "Start/Stop" without one of them needing to use uncommon keys, though "Start/Stop" may be better to bind to spacebar.